src/python/grpcio/grpc/__init__.py

```diff
@@ -858,6 +858,21 @@ def set_details(self, details):
     """
         raise NotImplementedError()
 
+    @abc.abstractmethod
+    def abort(self, code, details):
```

Comment:
If we mean for this to be used in favor of `set_code` and `set_details`, place it earlier-in-the-file-than ("above") `set_code` and `set_details`.